modules/backend/behaviors/formcontroller/update.php

```diff
@@ -0,0 +1,55 @@
+<?php
+$formController = $this->getClassExtension(\Backend\Behaviors\FormController::class);
+$formConfig = $formController->getConfig();
+?>
+
+<?php Block::put('breadcrumb') ?>
+    <ul>
+        <li><a href="<?= Backend::url($formConfig->defaultRedirect) ?>"><?= e(trans($formConfig->name . '_plural')); ?></a></li>
+        <li><?= e($this->pageTitle) ?></li>
+    </ul>
+<?php Block::endPut() ?>
+
+<?php if (!$this->fatalError): ?>
+    <?= Form::open(['class' => 'layout']) ?>
+        <div class="layout-row">
+            <?= $this->formRender() ?>
+        </div>
+
+        <div class="form-buttons">
+            <div class="loading-indicator-container">
+                <button
+                    type="button"
+                    data-request="onSave"
+                    data-request-data="redirect:0"
+                    data-hotkey="ctrl+s, cmd+s"
+                    data-load-indicator="<?= e(trans('backend::lang.form.saving_name', ['name' => trans($formConfig->name)])); ?>"
+                    class="btn btn-primary">
+                    <?= e(trans('backend::lang.form.save')); ?>
+                </button>
+                <button
+                    type="button"
+                    data-request="onSave"
+                    data-request-data="close:1"
+                    data-hotkey="ctrl+enter, cmd+enter"
+                    data-load-indicator="<?= e(trans('backend::lang.form.saving_name', ['name' => trans($formConfig->name)])); ?>"
+                    class="btn btn-default">
+                    <?= e(trans('backend::lang.form.save_and_close')); ?>
+                </button>
+                <button
+                    type="button"
+                    class="wn-icon-trash-o btn-icon danger pull-right"
+                    data-request="onDelete"
+                    data-load-indicator="<?= e(trans('backend::lang.form.deleting_name', ['name' => trans($formConfig->name)])); ?>"
+                    data-request-confirm="<?= e(trans('backend::lang.form.confirm_delete')); ?>">
+                </button>
+                <span class="btn-text">
+                    or <a href="<?= Backend::url($formConfig->defaultRedirect) ?>"><?= e(trans('backend::lang.form.cancel')); ?></a>
+                </span>
```

Comment:
```suggestion
                </span>
                <?= $this->fireViewEvent('backend.form.extendUpdateView.buttons') ?>
```

Reply:
@mjauvin I'd prefer if this was handled the same way that the RelationController handles list toolbar buttons with specific partial names that can be used / a config item in the config_form.yaml.